wordpress-mobile/WordPress-iOS
# Use readable width for post settings screens

Refs #10664

This is a first pass at improving the post settings UI on iPad. This sets all the post settings screen to use `cellLayoutMarginsFollowReadableWidth`. I don't think this should close #10664 as I think there's a more in-depth design review in progress (@rachelmcr?).

### Some examples:

![simulator screen shot - ipad pro 11-inch - 2018-12-18 at 14 50 15](https://user-images.githubusercontent.com/517257/50188033-0c599d80-02d5-11e9-9f9a-b4167f20c9ba.png)
![simulator screen shot - ipad pro 11-inch - 2018-12-18 at 14 50 21](https://user-images.githubusercontent.com/517257/50188039-0f548e00-02d5-11e9-9a6a-4e33963ba967.png)
![simulator screen shot - ipad pro 11-inch - 2018-12-17 at 17 22 16](https://user-images.githubusercontent.com/517257/50188043-124f7e80-02d5-11e9-84a2-e8e01971aa7b.png)

To test:
- Open the post settings screen and tap around. 
- Ensure all the reachable screens are a readable width.

Update release notes:
